sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/DelegatingCounter.java

```diff
@@ -17,12 +17,11 @@
  */
 package org.apache.beam.sdk.metrics;
 
-// Intentionally package private, this is meant to be used with Metrics.java and
-// LabeledMetrics.java
-
 import java.io.Serializable;
+import org.apache.beam.sdk.annotations.Internal;
 
 /** Implementation of {@link Counter} that delegates to the instance for the current context. */
+@Internal
 public class DelegatingCounter implements Metric, Counter, Serializable {
```

Comment:
Can we make this package private like the comment said?

Reply:
Yes I tried to that I think and something failed, let me try again.

Reply:
Oh I found `runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/LabeledMetrics.java` needs it so that's why it cannot be package private.